en/reference/schema-reference.html

```diff
@@ -2481,6 +2486,15 @@ <h2 id="onnx-model">onnx-model</h2>
 </table>
 <p>For more details including examples, see <a href="../onnx.html">ranking with ONNX models.</a></p>
 
+<h2 id="significance">significance</h2>
+<p>
+Contained in <a href="#rank-profile">rank-profile</a>. True or false. By default this is false. When enabled Vespa will use the significance calculation based on provided significance models in the service.xml for the rank-profile it is defined in. 
+<pre>
+significance {
```

Comment:
This section will be extended with more configuration later (e.g. which formula to use to calculate the significance). Please restructure similar to https://docs.vespa.ai/en/reference/schema-reference.html#onnx-model.